pytroll_collectors/tests/test_segments.py

```diff
@@ -77,15 +76,12 @@ def __init__(self, data, message_type='file', subject='/foo/viirs'):
         self.subject = subject
 
 
-class TestSegmentGatherer(unittest.TestCase):
+@pytest.mark.usefixtures("caplog")
+class TestSegmentGatherer:
     """Tests for the segment gatherer."""
 
     @pytest.fixture(autouse=True)
-    def inject_fixtures(self, caplog):
-        """Inject fixtures."""
-        self._caplog = caplog
-
-    def setUp(self):
+    def setup(self):
```

Comment:
I think if you call this `setup_method`, then the fixture autouse command above could be removed.

Reply:
Seems so. Fixed in https://github.com/pytroll/pytroll-collectors/pull/133/commits/e727700a21ccfaeb0d12bd4b686c062b271fd8a5